### DISCUSSION TITLE
what's the default name of the pivot table in a manytomany relation?

### DISCUSSION THREAD

**Discussion by 90umut** [adonisjs/core#4645] [General]
"The default value for the pivot table name is computed by [combining](https://github.com/adonisjs/lucid/blob/develop/src/orm/naming_strategies/camel_case.ts#L74) the parent model name and the related model name. However, you can also define a custom pivot table."

but if I declare my relation in both models, the table name could be user_skill or skill_user....

**Comment by thetutlage** [comment]
Nope. Its sorted alphabetically

**Reply by 90umut** [reply]
thank you for your reply, can you just say me the utility of declaring in both models please?

**Reply by RomainLanz** [reply]
Please see https://lucid.adonisjs.com/docs/relationships#custom-pivot-table

**Reply by thetutlage** [reply]
https://github.com/adonisjs/lucid/blob/develop/src/orm/naming_strategies/camel_case.ts#L74
